tests/package.json

```diff
@@ -9,6 +9,7 @@
 		"test:unit:deno": "TEST=true deno test unit/ --allow-all --unstable --import-map ../import_map.json",
 		"test:unit:jest": "TEST=true jest --config jest.config.unit.ts --coverage --no-cache",
 		"test:unit:deno:debug": "TEST=true deno test unit/ --allow-all --unstable --inspect-brk",
+		"test:unit:protocol": "TEST=true deno test unit/taqueria-protocol --allow-all --unstable --inspect-brk",
```

Comment:
Is this really needed for the VSCode tests?

Reply:
No, I've removed it.